arrow-core/src/main/kotlin/arrow/core/extensions/sequenceK.kt

```diff
@@ -226,13 +259,19 @@ interface SequenceKOrder<A> : Order<SequenceK<A>> {
       }.value()
 }
```

Comment:
We can't forget to deprecate these all later.

Reply:
Yes, I didn't want to add deprecate these instances because you're deprecating the typeclasses in separated pull requests. :)